tests/wicked/advanced_ref.pm

```diff
@@ -34,29 +33,35 @@ sub create_tunnel_with_commands {
 sub run {
     my ($self) = @_;
 
-    record_info('Test 1', 'Create a gre interface from legacy ifcfg files');
+    record_info('Test 1', 'Create a GRE interface from legacy ifcfg files');
     $self->create_tunnel_with_commands("gre1", "gre", "24");
     mutex_wait('test_1_ready');
-    assert_script_run("ifdown gre1");
-    # Placeholder for Test 2: Create a GRE interface from Wicked XML files
+    assert_script_run("ip addr flush dev gre1");
+    assert_script_run("ip tunnel delete gre1");
```

Comment:
you could also group this commands in a cleanup function

Reply:
will do this after I will get a green bubble with what I have now :)